Tests/AppTests/PackageCollectionTests.swift

```diff
@@ -769,3 +772,120 @@ class PackageCollectionTests: AppTestCase {
         )
     }
 }
+
+
+class SigningTests: XCTestCase {
+
+    func test_sign_collection() throws {
+        try XCTSkipIf(!isRunningInCI && Current.collectionSigningPrivateKey() == nil, "Skip test for local user due to unset COLLECTION_SIGNING_PRIVATE_KEY env variable")
```

Comment:
We don't want this to fail for someone running the tests locally if they haven't got the private key set up, so we skip in that case. But we _don't_ skip if it's running in CI, in which case the key must be set up and the test should be run.